translator/cmdutil/translatorutil.go

```diff
@@ -224,7 +225,32 @@ func TranslateJsonMapToYamlConfig(jsonConfigValue interface{}) (interface{}, err
 	if err = conf.Marshal(cfg); err != nil {
 		return nil, err
 	}
-	return conf.ToStringMap(), nil
+	strMap := conf.ToStringMap()
+	RemoveTLSRedacted(strMap)
+	return strMap, nil
+}
+
+// TODO: Switch to a custom translator approach at some point
```

Comment:
nit: not sure how helpful this is -- would it be better to have a short comment on what we are doing instead?